code/modules/mob/living/carbon/life.dm

```diff
@@ -100,9 +100,13 @@
 	if((status_flags & GODMODE))
 		return
 
+	var/lungs = getorganslot("lungs")
+	if(!lungs)
+		adjustOxyLoss(2)

```

Comment:
Won't this mean that Xenos are going to take damage, or do they have an override somewhere?